### PR TITLE
[KEYCLOAK] - Updating guide for multi-tenant deployments

### PR DESCRIPTION
**Pull request by pedroigor**

### CODE REVIEW

File: docs/src/main/asciidoc/keycloak-guide.adoc [hunk 1]
```diff
@@ -334,8 +334,51 @@ export access_token=$(\
 )
 ```
 
+== Multi-Tenant Deployments
+
+The Keycloak Quarkus Extension also supports multitenancy. This capability is specially useful if your service is serving different
```
Comment (gsmet) [inline docs/src/main/asciidoc/keycloak-guide.adoc]: we should be consistent about how `multitenancy` is spelled.

**Review by gsmet — COMMENTED** [review]
I added a very very minor comment, otherwise it looks good, thanks!

**Review by gsmet — APPROVED** [review]

File: docs/src/main/asciidoc/keycloak-guide.adoc (hunk 1 above)
Comment (pedroigor) [reply docs/src/main/asciidoc/keycloak-guide.adoc]: Good point. I change all references to `multitenancy`.

**Comment by gsmet** [comment]
Merged, thanks @pedroigor

**Pull request merged**